.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - 'v*.*.*' # Enforce Semantic Versioning
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+  
+    steps: 
+    - name: Setup CD Environment
+      uses: yetanalytics/actions/setup-env@v0
+    
+    - name: Extract version
+      id: version
+      run: echo ::set-output name=VERSION::${GITHUB_REF#refs\/tags\/v}
+
+    - name: Build JAR file
+      run: make build VERSION=${{ steps.version.outputs.VERSION }}
+    
+    - name: Publish artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: colossal-squuid-artifact-${{ steps.version.outputs.VERSION }}
+        path: target/colossal-squuid-${{ steps.version.outputs.VERSION }}.jar
+
+    - name: Deploy to Clojars
+      run: make deploy VERSION=${{ steps.version.outputs.VERSION }}
+      env:
+        CLOJARS_USERNAME: ${{ secrets.CLOJARS_USERNAME }}
+        CLOJARS_PASSWORD: ${{ secrets.CLOJARS_PASSWORD }}
```

Comment:
Assuming that `CLOJARS_PASSWORD` here is a deploy token? https://clojars.org/tokens

Reply:
Yes

Reply:
👍